### PR TITLE
Remove env.development.presets from package.json

### PR DESCRIPTION
**Pull request by tannerwelsh**
See @tannerwelsh or @jeffreywescott for more details on the pain here

### CODE REVIEW

**Comment by heyheyjp** [comment]
@tannerwelsh: can we get a 1-2 line summary?

**Comment by tannerwelsh** [comment]
Meteor was trying to compile with babel using an older version that didn't recognize the configuration method currently in `package.json`, so it was breaking the build. This was the way I solved (read: worked around) it on my machine, although I know @jeffreywescott did some more investigating and found some more info on the problem.

**Comment by jeffreywescott** [comment]
Here's what's going on, though I'll admit my understanding of it is only ~ 80%.

**TL;DR** -- Meteor dependency management pre-1.3 is terrible. We're on 1.2.

**More detail:**
- Meteor has an `ecmascript` package that allows ES6+ constructs to be used. We depend on it for our packages, and many of the Rocket.Chat packages also depend on it.
- Said `ecmascript` ships with its own (somewhat dated) version of Babel (not sure exactly which).
- We added a `package.json`file at the top-level of the echo-chat repository in order to (a) make echo-chat "feel" similar to our other services (e.g., `npm start`, `npm run lint`, `npm test`, etc.), and (b) to make the codeclimate integration smoother. Part of this included moving our `.babelrc` into a `babel` section of `package.json`, and setting-up some `test`-environment-specific directives. I also added `development`-environment-specific directives, thinking that they would be necessary to run lint in development.
- During the Meteor "build" process in `development` (but not in `production`, obviously), those `development`-environment-specific directives caused Meteor's `ecmascript` package to barf for some reason, perhaps because of the dated Babel version. Removing them makes things work in development, and linting still works, too.

**Comment by heyheyjp** [comment]
Thanks both!

**Comment by tannerwelsh** [comment]
What a head-scratcher. Thanks for the summary @jeffreywescott. I'd love to dig deeper into this mystery, but I'm scared that I'll get eaten by the swamp monsters in Meteor land.
